integration/kubernetes/helm-chart/alluxio/CHANGELOG.md

```diff
@@ -145,4 +145,9 @@
 - Add a table of keys and default values for the Helm templates in README
 
 0.6.13
+
 - Add remote logger for Alluxio services, putting the centralized logs in emptyDir/hostPath/PVC
+
+0.6.14
+
+- Migrate master StatefuleSet and worker DaemonSet securityContext to Pod-level (see Issue #13096)
```

Comment:
nit: You can just put the link "https://github.com/Alluxio/alluxio/issues/13096" instead of `see Issue #blah`